internal-packages/workflow-designer-ui/src/editor/properties-panel/text-generation-node-properties-panel/model/search-domain-filter-panel-enhanced.tsx

```diff
@@ -0,0 +1,85 @@
+import { useEffect, useMemo } from "react";
```

Comment:
Is this a sample code?
If you don't need this file, I would appreciate it if you could delete it.